src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebResponseObject.Common.cs

```diff
@@ -62,12 +62,7 @@ public long RawContentLength
         /// <summary>
         /// Gets the response body content as a <see cref="MemoryStream"/>.
         /// </summary>
-        public MemoryStream RawContentStream
-        {
-            get { return (_rawContentStream); }
-        }
-
-        private MemoryStream _rawContentStream;
+        public MemoryStream RawContentStream { get; protected set; }
```

Comment:
Why the change to `protected`?

Reply:
I followed the same convention as :
```cs
public byte[] Content { get; protected set; }
```
and
```cs
public string RawContent { get; protected set; }
```